maintainer/format/autopep8.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
here and in the other new .sh files: should we copy-paste the license header from `maintainer/CI/fix_style.sh`?

Reply:
yes probably